plugins/annotations.md

```diff
@@ -0,0 +1,227 @@
+keywords: annotation annotations
+---
+
+## Introduction
+The TinyMCE Annotation API provides the ability to add/modify/delete annotations, listen to text selection events, and retrieve all annotations with the same annotation name. The Annotations API is a part of the TinyMCE core and functions similar to the formatting APIs in TinyMCE core. 
+
+The primary value that the Annotations API provides is that it tags each annotation with a unique identifier(uid) which is accessible via `editor.annotator`. This highlights the annotated content and wraps it in annotation markers. These markers can either stay in the content or be removed on `getContent`, depending on the user configuration (persistent setting).
+
+## Using the Annotator Plugin
+
+To set up the TinyMCE Annotation plugin, perform the following procedure:
+
+### 1. Configure the Annotate Button
+
+       To configure the annotate button on your toolbar, make the following changes:
+
+       
+       ```js
+        setup: (ed) => {
+            ed.addButton('annotate-alpha', {
+              text: 'Annotate',
+              onclick: () => {
+                const comment = prompt('Comment with?');
+                ed.experimental.annotator.annotate('alpha', {
+                  comment
+                });
+                ed.focus();
+              },
+       ```
+> Note: The annotator plugin is still in its experimental stage, hence we are using 'experimental' in 'ed.experimental.annotator.annotate'. A user will see a *Using experimental API: annotator* warning on his console. Please ignore this warning, we are working on it.
+
+### 2. Registering the Annotator Plugin
+
+The annotator API supports multiple annotation functions. Each annotation function must be registered with the annotator `(editor.annotator)`. 
+
+```js
+ed.on('init', () => {
+       ed.experimental.annotator.register('alpha', {
+         persistent: true,
+         decorate: (uid, data) => {
+           return {
+             attributes: {
+               'data-mce-comment': alpha.comment
+             }
+           };
+         }
+       });
+     });
+```
+This will register an annotation with the name `alpha`. In our example, when an `alpha` is being added to the document, a span marker will be created with class `alpha` and a data attribute for the author.
+> Note: The data passed through here is the same as the data specified when calling the annotate API. `decorate` is used to turn the annotation data into a document object model (DOM) representation.
+The uid passed through to `decorate` is either the uid field in the data object (if it exists), or a randomly generated uid if it doesn't. Annotator will be responsible for putting the uid on the span. The user does not need to do that part.
+
+### 3. Making the Plugin Available
+
+For adding the annotate tool to the toolbar that is registered with `alpha` set the value of the toolbar to:
+
+```js
+toolbar: "annotate-alpha”
```

Comment:
That looks like a smart quote. I think it should be a standard double-quote character.